Tests/AnyCodableTests/AnyCodableTests.swift

```diff
@@ -2,50 +2,81 @@
-        XCTAssertEqual(AnyCodable(true), AnyCodable(true))
-        XCTAssertEqual(AnyCodable(2), AnyCodable(2))
-        XCTAssertEqual(AnyCodable(Int8(2)), AnyCodable(Int8(2)))
-        XCTAssertEqual(AnyCodable(Int16(2)), AnyCodable(Int16(2)))
-        XCTAssertEqual(AnyCodable(Int32(2)), AnyCodable(Int32(2)))
-        XCTAssertEqual(AnyCodable(Int64(2)), AnyCodable(Int64(2)))
-        XCTAssertEqual(AnyCodable(UInt(2)), AnyCodable(UInt(2)))
-        XCTAssertEqual(AnyCodable(UInt8(2)), AnyCodable(UInt8(2)))
-        XCTAssertEqual(AnyCodable(UInt16(2)), AnyCodable(UInt16(2)))
-        XCTAssertEqual(AnyCodable(UInt32(2)), AnyCodable(UInt32(2)))
-        XCTAssertEqual(AnyCodable(UInt64(2)), AnyCodable(UInt64(2)))
-        XCTAssertEqual(AnyCodable(Float(2)), AnyCodable(Float(2)))
-        XCTAssertEqual(AnyCodable(Double(2)), AnyCodable(Double(2)))
-        XCTAssertEqual(AnyCodable("hi"), AnyCodable("hi"))
-        XCTAssertEqual(AnyCodable(["hi": AnyCodable(2)]), AnyCodable(["hi": AnyCodable(2)]))
-        XCTAssertEqual(AnyCodable([AnyCodable("hi"), AnyCodable("there")]), AnyCodable([AnyCodable("hi"), AnyCodable("there")]))
-        XCTAssertEqual(AnyCodable(["hi":1]), AnyCodable(["hi":1]))
-        XCTAssertEqual(AnyCodable(["hi":1.2]), AnyCodable(["hi":1.2]))
-        XCTAssertEqual(AnyCodable(["hi"]), AnyCodable(["hi"]))
-        XCTAssertEqual(AnyCodable([1]), AnyCodable([1]))
-        XCTAssertEqual(AnyCodable([1.2]), AnyCodable([1.2]))
-        XCTAssertEqual(AnyCodable([true]), AnyCodable([true]))
-
-        XCTAssertNotEqual(AnyCodable(()), AnyCodable(true))
-    }
-
-    func testVoidDescription() {
-        XCTAssertEqual(String(describing: AnyCodable(Void())), "nil")
-    }
-
-    func testJSONDecoding() throws {
+    func test_equality() throws {
+        XCTAssertEqual(AnyCodable.null, AnyCodable.null)
+        XCTAssertEqual(AnyCodable.bool(true), AnyCodable.bool(true))
+        XCTAssertEqual(AnyCodable.int(2), AnyCodable.int(2))
+        XCTAssertEqual(AnyCodable.double(2), AnyCodable.double(2))
+        XCTAssertEqual(AnyCodable.string("hi"), AnyCodable.string("hi"))
+        XCTAssertEqual(AnyCodable.object(["hi": .int(2)]), AnyCodable.object(["hi": .int(2)]))
+        XCTAssertEqual(AnyCodable.array([.string("hi")]), AnyCodable.array([.string("hi")]))
+        XCTAssertEqual(AnyCodable.array([.int(1)]), AnyCodable.array([.int(1)]))
+        
+        XCTAssertNotEqual(AnyCodable.null, AnyCodable.bool(true))
+        XCTAssertNotEqual(AnyCodable.null, AnyCodable.int(2))
+        XCTAssertNotEqual(AnyCodable.int(4), AnyCodable.string("hi"))
+        XCTAssertNotEqual(AnyCodable.string("hi"), AnyCodable.array([.string("hi")]))
+        XCTAssertNotEqual(AnyCodable.object(["hi": .int(2)]), AnyCodable.object(["hi": .double(3)]))
+    }
+    
+    func test_inits() throws {
+        XCTAssertEqual(AnyCodable(()), AnyCodable.null)
+        XCTAssertEqual(AnyCodable(true), AnyCodable.bool(true))
```

Comment:
```suggestion
        let falseBool = false
        XCTAssertEqual(AnyCodable(()), AnyCodable.null)
        XCTAssertEqual(AnyCodable(true), AnyCodable.bool(true))
        XCTAssertEqual(AnyCodable(falseBool), AnyCodable.bool(false))
```
This hits the `AnyCodable(_ value: Bool)` constructor.

Reply:
This is the only constructor not hit by tests; I'll go ahead and commit this.